es5.js

```diff
@@ -208,8 +212,17 @@ module.exports = {
     'no-proto': 2,
     // disallow declaring the same variable more then once
     'no-redeclare': 2,
+    'no-restricted-properties': [
+      2,
+      {
+        'property': '__defineGetter__',
+        'message': 'Please use Object.defineProperty instead.',
+      },
+    ],
```

Comment:
`__defineSetter__`, `__lookupGetter__`, `__lookupSetter__` would also be a good to add.